exposed-version-catalog/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import org.jetbrains.exposed.gradle.configureMavenCentralMetadata
+import org.jetbrains.exposed.gradle.signPublicationIfKeyPresent
+
+plugins {
+    `version-catalog`
+    `maven-publish`
+    signing
+}
+
+group = "org.jetbrains.exposed"
+
+catalog {
+    versionCatalog {
+        val version: String by rootProject
+        
+        from(files("${rootProject.projectDir}/gradle/libs.versions.toml"))
+        
+        // Add all exposed modules to the catalog
+        rootProject.subprojects.forEach { subproject ->
+            if (subproject.plugins.hasPlugin("maven-publish") && subproject.name != name) {
+                val moduleName = subproject.name
+                if (moduleName.startsWith("exposed-")) {
+                    val shortName = moduleName.removePrefix("exposed-")
+                    library(shortName.replace("-", "."), "org.jetbrains.exposed:$moduleName:$version")
+                } else if (moduleName == "spring-transaction") {
+                    library("spring.transaction", "org.jetbrains.exposed:$moduleName:$version")
+                }
+            }
+        }
+    }
+}
+
+publishing {
```

Comment:
@jetbrains-junie Move this to `buildSrc` as `configurePublishingVersionCatalog`.